setup.py

```diff
@@ -63,6 +63,11 @@ def readme():
         'clickhouse': [
             'clickhouse-connect~=0.6.23',
         ],
+        'cratedb': [
+            'psycopg2==2.9.3',
+            'psycopg2-binary==2.9.3',
+            'sshtunnel==0.4.0',
+        ],
```

Comment:
If we have the choice, I'd go for psycopg3 right away, but sure it can happen on behalf of a later iteration, depending on your urgency to submit the patch to upstream.